### PR TITLE
Document device compliance and domain password policy

### PR DESCRIPTION
**Pull request by eketo-msft**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b0224bb](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/b0224bb6c249bf9b4e7bc1a01689aba1eed727dd)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b0224bb](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/b0224bb6c249bf9b4e7bc1a01689aba1eed727dd)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b0224bb](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/b0224bb6c249bf9b4e7bc1a01689aba1eed727dd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/beta/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/domain.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/domain.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903012138503093-4072%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=HYRyXC9s1GlW%2FyfULwtLNkDtlom53husYhHDWk5FJtA%3D&st=2019-03-01T21%3A37%3A37Z&se=2019-04-01T21%3A42%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9d07a5e](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/9d07a5edc4f58ff8afb003c560aed0a42fac5e89)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9d07a5e](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/9d07a5edc4f58ff8afb003c560aed0a42fac5e89)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/microsoftgraph/microsoft-graph-docs, working directory: W:\gktd-s, target branch: master, target branch commit: 61ee42108cb0a0067124a6987822012405e6f5bb, source branch: eketo-device-domain-update, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020209038003-4072%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ulbVwaLBaX%2BdoxVSCTiAcaJWxFkgU6RkQCRb5oGhAYM%3D&st=2019-03-02T02%3A04%3A59Z&se=2019-04-02T02%3A09%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a9a0182](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/a9a01821e68f557289b7a0621a0112cd410248f8)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a9a0182](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/a9a01821e68f557289b7a0621a0112cd410248f8)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/microsoftgraph/microsoft-graph-docs, working directory: W:\fllt-s, target branch: master, target branch commit: 61ee42108cb0a0067124a6987822012405e6f5bb, source branch: eketo-device-domain-update, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020238118263-4072%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=mDThQ4GsTbdUvOrqFfqpfDhQZsxvjhoio5H%2BcZlYbAI%3D&st=2019-03-02T02%3A34%3A05Z&se=2019-04-02T02%3A39%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[064ef87](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/064ef87396a4d9042b2a1f32700bab2b752bd0e3)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[064ef87](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/064ef87396a4d9042b2a1f32700bab2b752bd0e3)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[064ef87](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/064ef87396a4d9042b2a1f32700bab2b752bd0e3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/beta/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/user.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/beta/resources/user.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/domain.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/domain.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/user.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/user.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020239252886-4072%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ncmslwGTrl8So1MuXff4DWWIxw5up4y%2BIUiep4sFKhQ%3D&st=2019-03-02T02%3A39%3A59Z&se=2019-04-02T02%3A44%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[fd40eb8](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/fd40eb83046420b473c8dff888c5410cad681405)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[fd40eb8](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/fd40eb83046420b473c8dff888c5410cad681405)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[fd40eb8](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/fd40eb83046420b473c8dff888c5410cad681405)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/beta/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/user.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/beta/resources/user.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/domain.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/domain.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/user.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/user.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903020247587064-4072%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JBZ5AmLtGs5wB6KEB8fL3JZLbLtRsFeN42VNKESw404%3D&st=2019-03-02T02%3A46%3A55Z&se=2019-04-02T02%3A51%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/v1.0/resources/user.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ This resource supports:
 |id|String|The unique identifier for the user. Inherited from [directoryObject](directoryobject.md). Key. Not nullable. Read-only.|
 |imAddresses|String collection|The instant message voice over IP (VOIP) session initiation protocol (SIP) addresses for the user. Read-only.|
 |interests|String collection|A list for the user to describe their interests.|
+|isResourceAccount|Boolean| **true** if the user is a resource account; otherwise, **false**. Null value should be considered **false**.|
```
Comment (dkershaw10) [inline api-reference/v1.0/resources/user.md]: I think this needs a bit more.  And the language is a bit weird - "if this user is a resource account" - although I'm actually struggling to come up with something better than - if this object is a resource account.  I'd also love us to provide some examples of a resource account.  I'm assuming something like a "conference room"?

Comment (eketo-msft) [reply api-reference/v1.0/resources/user.md]: I've reached out to get good examples of this.  Let's me finish this PR and update it so it does not block migration.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9a4e9fc](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/9a4e9fc204fe1760605949d080b261bb3d8435d2)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9a4e9fc](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/9a4e9fc204fe1760605949d080b261bb3d8435d2)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9a4e9fc](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4072/commits/9a4e9fc204fe1760605949d080b261bb3d8435d2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/beta/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/user.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/beta/resources/user.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/domain.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/domain.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/user.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/api-reference/v1.0/resources/user.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-domain-update/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903042354136282-4072%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=qMyP%2Fs9gwE5haqqA1JoIa%2BtTKg80rXfZvH5P7TplrpA%3D&st=2019-03-04T23%3A53%3A08Z&se=2019-04-04T23%3A58%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**